mandatory/3-playing-computer.js

```diff
@@ -28,7 +28,6 @@ const f2 = function (a, b) {
 
```

Comment:
I think we have to write the answers in the comments

Reply:
Thank you Daniel, I`ll write the answers.